CliFx.Tests/CliApplicationTests.cs

```diff
@@ -238,7 +239,8 @@ public async Task RunAsync_Cancellation_Test()
             // Arrange
             using (var stdoutStream = new StringWriter())
             {
-                var console = new VirtualConsole(stdoutStream);
+                var cancellationTokenSource = new CancellationTokenSource();
```

Comment:
`CancellationTokenSource` can be also put inside `using ()`

Reply:
Fixed.